source/arriving/whats_next.rst

```diff
@@ -0,0 +1,35 @@
+.. _ref_whats_next:
+
+.. role:: bash(code)
+   :language: bash
+
+What's next
+===========
+
+Now that you have set up your computer with the common SCIL setup, you are ready to take the next steps. There are probably five of them, with equal importance:
+
+    * | **Install other tools**
+      | Indeed, you have installed the bare minimum to function properly in the lab. There might be :ref:`ref_other_tools` useful to your work in the lab, like visualization tools. Ask your supervisor which tools to install among the following list:
+
+        * :ref:`ref_mi_brain`
+        * :ref:`ref_mrtrix`
+        * :ref:`ref_ants`
+        * :ref:`ref_fsl`
+        * :ref:`ref_freesurfer`
+        * :ref:`ref_dmri_explorer`
+        * :ref:`ref_dicom_converters`
+        * :ref:`ref_mricrogl`
+
+      | Follow the links and see the installation steps for the tools you want.
+
+    * | **Become familiar with the tools**
+      | The Tools section is a great resource to learn more about all the tools you have installed and more. For instance, it is where you can learn about our multiple :ref:`ref_flow` like :ref:`ref_tractoflow`. Don't forget to go back to this section whenever you forget something about a tool.
+
+    * | **Improve your knowlegde with our guides**
+      | The Guides section provides additionnal information on various subjects. It is a useful resource to gather more knowlegde about :ref:`ref_git`, :ref:`ref_python`, :ref:`ref_terminal`, :ref:`ref_bash` or :ref:`ref_anat`, to name a few important ones.
+
+    * | **Start reading about your project**
```

Comment:
which makes me think, we might consider having a bibliography management section (I played a bit with this lately so I could do it... after the CDMRI deadline).

Reply:
Very good idea!